### DISCUSSION TITLE
[FTL] Error, no templates were found. issues in fuzzing,workflows,miscellaneous

### DISCUSSION THREAD

**Discussion by d4715** [projectdiscovery/nuclei-templates#1765] [Q&A] [answered]
hello there, 

Today I update nuclei tool and templates in my kali linux in vmware version 14.1.3 build-9474260 (latest and updated )

kali linux details:
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2021.2"
VERSION_ID="2021.2"
VERSION_CODENAME="kali-rolling"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="https://www.kali.org/"
SUPPORT_URL="https://forums.kali.org/"
BUG_REPORT_URL="https://bugs.kali.org/"

but when i just testing some target that time [FTL] Error, no templates were found
occurred in miscellaneous,workflows,fuzzing
 
1.txt is my domains file 
see this image 
![1](https://user-images.githubusercontent.com/11803332/123097489-d1102a80-d44d-11eb-9031-2ff339c4d462.JPG)

now i describe here one by one. 
1) fuzzing,
when i run my script fuzzingtoday.sh that time i got [FTL] Error, no templates were found

1.1) my  fuzzingtoday.sh script image 
![2](https://user-images.githubusercontent.com/11803332/123097661-f8ff8e00-d44d-11eb-9e83-f7fd3c16675f.JPG)

1.2) screen shot of [FTL] Error, no templates were found in fuzzing 
![3](https://user-images.githubusercontent.com/11803332/123097914-3106d100-d44e-11eb-8302-046f1c1bd620.JPG)

2) workflows 
when i run my script workflowstoday.sh that time i got [FTL] Error, no templates were found but some templates are working 

2.1) my  workflowstoday.sh script image (I used -w for workflow you can see in this image)
![4](https://user-images.githubusercontent.com/11803332/123098246-9064e100-d44e-11eb-8a41-6f910164e50e.JPG)

2.2) screen shot of [FTL] Error, no templates were found in workflows 
![5](https://user-images.githubusercontent.com/11803332/123099196-96a78d00-d44f-11eb-886d-ea7fa0b6713b.JPG)

3)miscellaneous
when i run my script miscellaneoustoday.sh that time i got [FTL] Error, no templates were found. 

3.1)my  miscellaneoustoday.sh script image 
![6](https://user-images.githubusercontent.com/11803332/123100202-8512b500-d450-11eb-8f08-ca8f0159fbbe.JPG)

3.2)screen shot of [FTL] Error, no templates were found in miscellaneous 
![7](https://user-images.githubusercontent.com/11803332/123100470-c86d2380-d450-11eb-9b5f-560d693064bf.JPG)

so please help me where i done mistake.

Thanks and Regards, 
D4715

**Comment by geeknik** [comment, marked as answer]
Hi, please see the [Template Exclusion](https://nuclei.projectdiscovery.io/nuclei/get-started.html#template-exclusion) section for more information on this. This issue is most likely because [nuclei ignores](https://github.com/projectdiscovery/nuclei-templates/blob/master/.nuclei-ignore) certain classes of template out of the box.  For fuzzing, add `-tags fuzz` to your command line. For misc, add `-tags misc`. [Et al](https://nuclei.projectdiscovery.io/nuclei/get-started.html#running-nuclei).  Good luck out there! \m/

**Reply by d4715** [reply]
Hello There, 
Thanks for solutions . if any doubts occurred i will ping you back .
Thanks & Regards,
D4715
